Purchases.xcodeproj/project.pbxproj

```diff
@@ -1331,6 +1320,24 @@
 			path = APITester;
 			sourceTree = "<group>";
 		};
+		B3DDB55726854850008CCF23 /* Public */ = {
+			isa = PBXGroup;
+			children = (
+				B3DDB55A26854A0A008CCF23 /* ReOrgLater */,
+				B3D3C4712685784800CB3C21 /* Package.swift */,
+				B3DDB55826854865008CCF23 /* PurchaseOwnershipType.swift */,
+			);
+			path = Public;
+			sourceTree = "<group>";
+		};
+		B3DDB55A26854A0A008CCF23 /* ReOrgLater */ = {
```

Comment:
Why `ReOrgLater`?

Reply:
Since they are `EntitlementInfo` and `EntitlementInfos` related, I was going to wait to address that, then organize